docs/index.md

```diff
@@ -5,42 +5,37 @@ nav_order: 1
+* [**Userland components validate each other:**](binaries/index.md) Each of the userland components (the daemon, the GUI agent, and the command-line utility) communicate with each other using XPC and check that their signing certificates are identical before any communication is accepted.
+* **Caching:** Allowed binaries are cached so the processing required to make a request is only done if the binary isn't already cached.
 
-* [Configuration](deployment/configuration.md): The local and sync server configuration options, along with example needed mobileconfig files.
-* [Troubleshooting](deployment/troubleshooting.md): While there are numerous pages with details on Santa, admins may appreciate a central place to branch off from with common practical issues.
+## Documentation overview
 
-#### Development
+### Introduction
 
-* [Building Santa](development/building.md): How to build and load Santa for testing on a development machine.
-* [Contributing](development/contributing.md): How to contribute a bug fix or new feature to Santa.
+The following pages give an overview of how Santa accomplishes binary authorization at the enterprise scale.
 
-#### Details
-
-For those who want even more details on how Santa works under the hood, this section is for you.
-
-###### Binaries
+* [Binary Authorization](introduction/binary-authorization-overview.md): How Santa makes allow or deny decisions for any `execve()` taking place.
+* [Syncing](introduction/syncing-overview.md): How configuration and rules are applied from a sync server.
 
-There are five main components that make up Santa whose core functionality is described in snippets below. For additional detail on each component, visit their respective pages. These quick descriptions do not encompass all the jobs performed by each component, but do provide a quick look at the basic functionality utilized to achieve the goal of binary authorization.
+### Deployment
 
-* [santad](details/santad.md): A user-land root daemon that makes decisions.
-* [santactl](details/santactl.md): A user-land anonymous daemon that communicates with a sync server for configurations and policies. santactl can also be used by a user to manually configure Santa when using the local configuration.
-* [santa-gui](details/santa-gui.md): A user-land GUI daemon that displays notifications when an `execve()` is blocked.
-* [santabs](details/santabs.md): A user-land root daemon that finds Mach-O binaries within a bundle and creates events for them.
+* [Configuration](deployment/configuration.md): The local and sync server configuration options, along with example needed mobileconfig files.
+* [Troubleshooting](deployment/troubleshooting.md): While there are numerous pages with details on Santa, admins may appreciate a central place to branch off from with common practical issues.
```

Comment:
I agree with the premise of the troubleshooting page but would it not be clearer to say something like:

How to troubleshoot issues with your Santa deployment